src/main/java/run/halo/starter/StarterPlugin.java

```diff
@@ -0,0 +1,31 @@
+package run.halo.starter;
+
+import org.springframework.stereotype.Component;
+import run.halo.app.plugin.BasePlugin;
+import run.halo.app.plugin.PluginContext;
+
+/**
+ * <p>Plugin main class to manage the lifecycle of the plugin.</p>
+ * <p>This class must be public and have a public constructor.</p>
+ * <p>Only one main class extending {@link BasePlugin} is allowed per plugin.</p>
+ *
+ * @author guqing
+ * @since 1.0.0
+ */
+@Component
+public class StarterPlugin extends BasePlugin {
```

Comment:
建议将类名改为 `StickerPlugin`
```suggestion
public class StickerPlugin extends BasePlugin {
```